data/json/items/archery.json

```diff
@@ -184,32 +184,76 @@
       "damage" : 14,
       "pierce" : 7,
       "range" : 10,
-      "dispersion" : 60,
+	  "dispersion": 80,
       "loudness": 0,
       "count" : 10,
-      "effects" : ["RECOVER_15"]
+	  "effects": ["RECOVER_50"]
    },
+   {
+   	  "type": "AMMO",
+	  "id": "arrow_metal_bodkin",
+	  "price": 4000,
+	  "name": "metal bodkin arrow",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "This metal arrow has fletching and a steel bodkin arrowhead for armor penetration.  Stands a very good chance of remaining intact once fired.",
+	  "material": "steel",
+	  "volume": 1,
+	  "weight": 60,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "arrow",
+	  "damage": 10,
+	  "pierce": 15,
+	  "range": 12,
+	  "dispersion": 75,
+	  "loudness": 0,
+	  "count": 10,
+	  "effects": ["RECOVER_50"]
+   },
+   {
+	  "type": "AMMO",
+	  "id": "arrow_metal_target",
+	  "price": 4000,
+	  "name": "metal target arrow",
+	  "symbol": "=",
+	  "color": "green",
+	  "description": "This metal arrow has extra long fletching and a round, balanced arrowhead for extra accuracy, at the cost of some damage.  Stands a very good chance of remaining intact once fired.",
+      "material": "steel",
+	  "volume": 1,
+	  "weight": 50,
+	  "bashing": 3,
+	  "cutting": 2,
+	  "ammo_type": "arrow",
+	  "damage": 12,
+	  "pierce": 5,
+	  "range": 8,
+	  "dispersion": 50,
+	  "loudness": 0,
+	  "count": 10,
+	  "effects": ["RECOVER_60"]
+   }, 
    {
       "type" : "AMMO",
       "id" : "arrow_cf",
       "price" : 9500,
       "name" : "carbon fiber arrow",
       "symbol" : "=",
       "color" : "green",
-      "description" : "High-tech carbon fiber shafts and 100 grain broadheads.  Very lightweight, fast, and notoriously fragile.  Stands a bad chance of remaining intact once fired.",
```

Comment:
Spaces between `"description" :` should be reverted for styling. Similarly, two spaces between sentences (this should be in [JSON_STYLE.md](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/JSON_STYLE.md) but looks like that's missing), not just one.

Reply:
Will do. Weird though.

Reply:
Fixed.